rh-che #707: Disable organizations on rh-che

amisevsk opened the pull request:
### What does this PR do?
Updates messages given when user navigates to 'Organizations' tab [1] and workspace 'Sharing' tab [2] to explain that these features are unavailable.

#### More detail

Add new module to maven build: fabric8-ide-dashboard-war, that modifies the upstream dashboard build to override files so that we can change upstream in a convenient and simple way.

Modifies the organizations page and share-workspace page to give informative messages about how these features are currently unavailable on osio.

Building the dashboard (which is an angularjs project) is nonstandard, so there is detailed documentation in ./assembly/fabric8-ide-dashboard-war/README.md

### What issues does this PR fix or reference?
#707

### How have you tested this PR?
Manually on the dev cluster. Image built off this PR is available at `amisevsk/rhche-server:issue-707`

[1] - ![screenshot from 2018-08-28 14-34-46](https://user-images.githubusercontent.com/16168279/44749976-297a1e80-aae2-11e8-9da1-ac7fd080f71e.png)
[2] - ![screenshot from 2018-08-28 14-34-59](https://user-images.githubusercontent.com/16168279/44749986-2da63c00-aae2-11e8-8ee0-dde28170d8d3.png)

ibuziuk commented on assembly/fabric8-ide-dashboard-war/src/app/workspaces/share-workspace/share-workspace.html:
```diff
@@ -0,0 +1,17 @@
+
+-->
+<div class="no-permissions-error" layout="row" layout-align="center center">
+  <span class="material-design icon-ic_error_outline_24px"></span>
```
@amisevsk workspace sharing should still be enabled but with some limitations - https://github.com/redhat-developer/rh-che/issues/798. So, in this PR I would propose just to disable organizations

amisevsk replied:
Good point. Done.

amisevsk commented:
@davidfestal I'm not sure if there's a convenient way to do this, while keeping the diff readable -- I will look into it though. Do you think something like using [verbose](https://ant.apache.org/manual/Tasks/copy.html) and logging to a file in the ant task would be sufficient? A plain diff of the two source trees will result in hundreds of lines of `-`, as there are far fewer files in the `./src` directory in general.

amisevsk commented:
@davidfestal I've added some logging to the build process:
1. A diff is taken between `./src` and `./target/sources/src` and saved to file `source_tree.diff` in the `target` dir
2. Added the verbose flag to the copy command, so that it at least lists how many files are copied.

WDYT?

davidfestal commented:
> @davidfestal I've added some logging to the build process:
> - A diff is taken between ./src and ./target/sources/src and saved to file source_tree.diff in the target dir
> - Added the verbose flag to the copy command, so that it at least lists how many files are copied.
>
> WDYT?

That seems good to me
